Automattic/sensei
# Merge feature branch: onboarding theme step

Fixes #6195 

Now that all of the PR's are completed for the Course Theme step in the onboarding wizard, let's merge it into `trunk`!

### Testing instructions

Go through the Setup Wizard for Sensei and test the Course Theme step to ensure it is working as expected.
